src/components/Input/InputTypes.js

```diff
@@ -0,0 +1,22 @@
+const InputTypes = [
+  "color",
+  "date",
+  "datetime-local",
+  "email",
+  "hidden",
+  "month",
+  "number",
+  "password",
+  "radio",
+  "range",
+  "reset",
+  "search",
+  "submit",
+  "tel",
+  "text",
+  "time",
+  "url",
+  "week"
```

Comment:
`file` type?

Reply:
That's going to be a separate component of its own. Have removed type checkbox radio as well since both of them are their own separate components with different markup and logic.